<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SPRSpectrumListTableWidget</class>
 <widget class="QWidget" name="SPRSpectrumListTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1161</width>
    <height>778</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1020</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSplitter::handle:horizontal {
   width: 6px;
	background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(242, 242, 242), stop:1 rgb(149, 150, 149))
}

QSplitter::handle:vertical {
    height: 6px;
	background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(149, 150, 149), stop:1 rgb(242, 242, 242))
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="spTables">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="w_Up" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="spElements">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QGroupBox" name="gbSpectrumElements">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Области спектра</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="SPRZonesRangesTableWidget3" name="tRangesChannel" native="true"/>
           </item>
           <item>
            <widget class="QGroupBox" name="gbViewAsIs">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Отображенние (как есть)</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>3</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QRadioButton" name="rbScales">
                <property name="text">
                 <string>Масштабировать</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rbAsOneSecond">
                <property name="text">
                 <string>Приводить к 1 секунде</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="SPRSpectrumGraphicsWidget" name="graphic" native="true">
          <property name="minimumSize">
           <size>
            <width>750</width>
            <height>300</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="w_Down" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>44</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="bRecompliteSpectrums">
            <property name="text">
             <string>Пересчитать спектры</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/b_refresh_green.png</normaloff>:/icons/icons/b_refresh_green.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bDeleteSpectors">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Удалить спектры</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/b_Cancel.png</normaloff>:/icons/icons/b_Cancel.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bGetSpectrums">
            <property name="text">
             <string>Снять</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/b_Get_Spectrum.png</normaloff>:/icons/icons/b_Get_Spectrum.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bOpenSpectrums">
            <property name="text">
             <string>Открыть</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/b_Open.png</normaloff>:/icons/icons/b_Open.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bSaveSpectrums">
            <property name="text">
             <string>Сохранить</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/b_Save.png</normaloff>:/icons/icons/b_Save.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bSeparatorOff">
            <property name="text">
             <string>Выкл. сортировочную машину</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/b_Off.png</normaloff>:/icons/icons/b_Off.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bAverage">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Усреднить</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/icons/b_Average.png</normaloff>:/icons/icons/b_Average.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbBasetSpectrumVisible">
            <property name="text">
             <string>Базовые спектры</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lSpectrumFName">
            <property name="text">
             <string>Имя файла спектров</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSpectrumsFName"/>
          </item>
          <item>
           <widget class="QToolButton" name="bSpectrumFNameSelect">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSplitter" name="spBaseSpectrums">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="childrenCollapsible">
             <bool>false</bool>
            </property>
            <widget class="SPRSpectrumListTable2" name="tListSpectrumItem">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
            </widget>
            <widget class="QGroupBox" name="gbBasetSpectrums">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string>Базовые спектры</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QWidget" name="widget_5" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
                 <item>
                  <widget class="SPRSpectrumBaseListTable3" name="tListBasedSpectrumItem">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QWidget" name="widget_6" native="true">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <property name="leftMargin">
                  <number>9</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="bGetBaseSpectrum">
                   <property name="text">
                    <string>Получить базовые спектры</string>
                   </property>
                   <property name="icon">
                    <iconset resource="icons.qrc">
                     <normaloff>:/icons/icons/b_Get_Base_Spectrum.png</normaloff>:/icons/icons/b_Get_Base_Spectrum.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>32</width>
                     <height>32</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SPRSpectrumGraphicsWidget</class>
   <extends>QWidget</extends>
   <header>sprspectrumgraphicswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRZonesRangesTableWidget3</class>
   <extends>QWidget</extends>
   <header>sprzonesrangestablewidget3.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRSpectrumListTable2</class>
   <extends>QTableWidget</extends>
   <header>sprspectrumlisttable2.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SPRSpectrumBaseListTable3</class>
   <extends>QTableWidget</extends>
   <header>sprspectrumbaselisttable3.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cbBasetSpectrumVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbBasetSpectrums</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>610</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
